src/olympia/addons/serializers.py

```diff
@@ -497,7 +514,7 @@ class ESAddonSerializer(BaseESSerializer, AddonSerializer):
     # data the same way than the regular serializer does (usually because we
     # some of the data is not indexed in ES).
     authors = BaseUserSerializer(many=True, source='listed_authors')
-    current_version = SimpleESVersionSerializer()
```

Comment:
I guess it's too much to hope for that `SimpleESVersionSerializer` is now unused?